src/Mod/Start/StartPage/StartPage.py

```diff
@@ -245,27 +234,27 @@ def buildCard(filename,method,arg=None):
     """builds an html <li> element representing a file. 
     method is a script + a keyword, for ex. url.py?key="""
 
-    result = encode("")
+    result = ("")
```

Comment:
The parentheses can be removed here.

Reply:
Done a07c5e987873995278d2cc986e6539206e8a664c